### ISSUE TITLE
[WASM] TF.js WASM backend

### ISSUE THREAD

**Issue by akineeic** [intel/webml-polyfill#1070]
TF.js wasm backend seems to be in the early stage of development, only a few ops are supported, even Conv2D was implemented 5 days ago in this [commit](https://github.com/tensorflow/tfjs/commit/ea83eee6a65e4221add99654509576e4c2169f9e#diff-e04b82012b4d6362f0895abb62d58164). Many common layers like POOL_2D and DEPTHWISE_CONV_2D is not supported, so there is no model used in polyfill can be run in TF.js wasm backend now. The ops not supported will be updated in the next comment.

**Comment by akineeic** [comment]
- <h4>Ops gap
The table below records the ops which have not been implemented in TF.js WASM backend but used in polyfill example.

Ops | Model
------------ | -------------
AVERAGE_POOL_2D | MobileNet_v1, MobileNet_v1_Quant, MobileNet_v2, MobileNet_v2_Quant, SqueezeNet, Inception_v3, Inception_v3_Quant, Inception_v4, Inception_v4_Quant, Inception_ResNet_v2, all DeepLab
BATCH_TO_SPACE_ND | SRGAN
DEPTHWISE_CONV_2D | MobileNet_v1, MobileNet_v1_Quant, MobileNet_v2, MobileNet_v2_Quant, all SSD_MobileNet, all DeepLab
FULLY_CONNECTED | Inception_v4, Inception_v4_Quant, Inception_ResNet_v2
MAX_POOL_2D | SqueezeNet, Inception_v3, Inception_v3_Quant, Inception_v4, Inception_v4_Quant, Inception_ResNet_v2, Tiny_Yolo_v2, KWS_CNN
MAXIMUM | Tiny_Yolo_v2
RESHAPE | MobileNet_v1_Quant, MobileNet_v2, MobileNet_v2_Quant, SqueezeNet, Inception_v3, Inception_v3_Quant, KWS_CNN
RESIZE_BILINEAR| all DeepLab
SOFTMAX | MobileNet_v1, MobileNet_v1_Quant, MobileNet_v2, SqueezeNet, Inception_v3, Inception_v4, Inception_v4_Quant
SQUEEZE | MobileNet_v1
TANH | SRGAN

**Comment by akineeic** [comment]
Since there are some ops  that should have been added into the wasm backend but are not available right now, I have created an [issue](https://github.com/tensorflow/tfjs/issues/2649) to report the bug. Waiting for their suggestion.

**Comment by akineeic** [comment]
With the update of tfjs-backend-wasm, I have successfully import the module to our project. Now I'm trying to convert the operands in WASM to tf tensor by refering the code in WebGL, and at the same time I'll consider how to merge the two backends to reduce the code.
